### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by danggui1995**
1.fix vmdl export exception
2.fix shader RegexInclude not match

### CODE REVIEW

**Comment by SinZ163** [comment]
A pull request is just merging branch A to branch B, you don't need to close and create new pull requests if you are trying to merge the exact same branch, it will update when you do more commits

**Comment by danggui1995** [comment]
Got it, thanks.

File: GUI/Types/Renderer/ShaderLoader.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class ShaderLoader
     {
         private const string ShaderDirectory = "GUI.Types.Renderer.Shaders.";
         private const int ShaderSeed = 0x13141516;
-        private static Regex RegexInclude = new Regex(@"^#include ""(?<IncludeName>[^""]+)""$", RegexOptions.Multiline);
+        private static Regex RegexInclude = new Regex(@"^#include ""(?<IncludeName>[^""]+)""", RegexOptions.Multiline);
         private static Regex RegexDefine = new Regex(@"^#define param_(?<ParamName>\S+) (?<DefaultValue>\S+)", RegexOptions.Multiline);
```
Comment (xPaw) [inline GUI/Types/Renderer/ShaderLoader.cs:19]: Why?

File: GUI/Types/Renderer/ShaderLoader.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ public class ShaderLoader
     {
         private const string ShaderDirectory = "GUI.Types.Renderer.Shaders.";
         private const int ShaderSeed = 0x13141516;
-        private static Regex RegexInclude = new Regex(@"^#include ""(?<IncludeName>[^""]+)""$", RegexOptions.Multiline);
```
Comment (xPaw) [inline GUI/Types/Renderer/ShaderLoader.cs:18]: Why?

Comment (danggui1995) [reply GUI/Types/Renderer/ShaderLoader.cs:18]: `GL.GetShader(vertexShader, ShaderParameter.CompileStatus, out var shaderStatus);`
Didn't you get the exception? I come to this exception and think the $ will not match when the next char is \r\n

Comment (xPaw) [reply GUI/Types/Renderer/ShaderLoader.cs:18]: I added an optional `\r` there.

Can you test latest master for both that and the exporting code? I changed it to use same code as the renderer so it doesn't need to be updated in two places.

Comment (danggui1995) [reply GUI/Types/Renderer/ShaderLoader.cs:18]: all tested ok

**Pull request closed without merging**